datadog/resource_datadog_synthetics_test_.go

```diff
@@ -49,12 +49,60 @@ func resourceDatadogSyntheticsTest() *schema.Resource {
 				Optional: true,
 			},
 			"assertions": {
-				Type:     schema.TypeList,
-				Optional: true,
+				Type:          schema.TypeList,
+				Optional:      true,
+				ConflictsWith: []string{"assertion"},
+				Deprecated:    "Use assertion instead",
 				Elem: &schema.Schema{
 					Type: schema.TypeMap,
 				},
 			},
+			"assertion": {
+				Type:          schema.TypeList,
+				Optional:      true,
+				ConflictsWith: []string{"assertions"},
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"type": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"operator": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"property": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"target": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"targetjsonpath": {
```

Comment:
Should we have this conflictsWith `target`? They seem mutually exclusive

Reply:
Good idea.

Reply:
Can we try to bump the SDK version to https://github.com/hashicorp/terraform-plugin-sdk/releases/tag/v1.15.0